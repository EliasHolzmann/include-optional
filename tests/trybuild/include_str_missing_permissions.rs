fn main() {
    include_optional::include_str_optional!("/dev/mem");
}