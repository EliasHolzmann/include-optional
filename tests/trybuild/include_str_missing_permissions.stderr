error: couldn't read /dev/mem: Permission denied (os error 13)
 --> $DIR/include_str_missing_permissions.rs:2:5
  |
2 |     include_optional::include_str_optional!("/dev/mem");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_str` (in Nightly builds, run with -Z macro-backtrace for more info)
